lib/bundler/plugin.rb

```diff
@@ -45,6 +45,26 @@ def install(names, options)
       Bundler.ui.error "Failed to install plugin #{name}: #{e.message}\n  #{e.backtrace.join("\n ")}"
     end
 
+    # List installed plugins and commands
+    #
+    def list
+      installed_plugins = index.installed_plugins
```

Comment:
i would add add `output = String.new` and then append to it to help make this more readable.